src/mcts/search.h

```diff
@@ -325,6 +325,12 @@ class SearchWorker {
   const bool moves_left_support_;
   IterationStats iteration_stats_;
   StoppersHints latest_time_manager_hints_;
+  // Intermediate array to store values when processing policy temperature decay.
+  // According to a lichess developer post:
+  // https://lichess.org/blog/Wqa7GiAAAOIpBLoY/
+  //     developer-update-275-improved-game-compression
+  // there are never more than 256 valid legal moves in any legal position.
+  float intermediate[256];
```

Comment:
This needs a trailing underscore in its name.

Reply:
Thanks, it is fixed now.